### PR TITLE
Enhance python documentation

### PR DESCRIPTION
**Pull request by judu**
This is WIP, since we need the configuration environment variables support for python to get in production first.

### CODE REVIEW

**Comment by judu** [comment]
Alright, the changes are in production, we can merge this if it pleases you.

File: runtimes/python/python_apps.md [hunk 1]
```diff
@@ -78,36 +74,19 @@ of your project with a Flask `app` object inside.
...
-      }
-   }
+CC_PYTHON_CELERY_MODULE="mymodule"
 ```
 
-The `log` path will be concatenated to the application's path.
+You can also activate beat with `CC_PYTHON_CELERY_USE_BEAT=true` and provide a given log
+dir for celery with `CC_PYTHON_CELERY_LOGFILE="/path/to/logdir"`.
+
+The `CC_PYTHON_CELERY_LOGFILE` path will be concatenated to the application's path.
```
Comment (divarvel) [inline runtimes/python/python_apps.md]: I'd rather have "the `CC_PYTHON_CELERY_LOGFILE` path is relative to the application path". It's more user centric.

File: runtimes/python/python_apps.md [hunk 2]
```diff
@@ -116,6 +95,12 @@ Currently, we support `uwsgi` and `gunicorn` for python backends. To select one,
 
 If not specified, the default backend is `uwsgi`.
 
+## Using the Gevent loop engine
+
+Whether you use uwsgi or gunicorn, you can enable the Gevent loop engine.
+
+To do so, just add the `CC_PYTHON_USE_GEVENT` environment variable to your application, with the `true` value.
```
Comment (divarvel) [inline runtimes/python/python_apps.md]: s/just//

File: runtimes/python/python_apps.md [hunk 3]
```diff
@@ -53,23 +53,19 @@ Remove the environment variable or set it to `false` to disable this feature.
 If you have any question, feel free to [contact our support](https://www.clever-cloud.com/doc/get-help/support/).
 
 
-## Configuration file
+## Configuring the deployment
```
Comment (divarvel) [inline runtimes/python/python_apps.md]: I'd like to use the term `deployment` less (as long as we've not properly defined it)

Comment (divarvel) [inline runtimes/python/python_apps.md]: Configuring your application conveys the same meaning while being a bit clearer

File: runtimes/python/python_apps.md [hunk 4]
```diff
@@ -53,23 +53,19 @@ Remove the environment variable or set it to `false` to disable this feature.
...
 
-The configuration file for your Python application must be `/clevercloud/python.json`.
-
-You can find a [example of Flask application](/doc/python/python-flask-sample-app/) who use concept detailed below.
 
+To make the deployment process aware of what you are trying to run, you will use some
+environment variables. To set such environment variables, go in the "Environment
+variables" menu of your application.
```
Comment (divarvel) [inline runtimes/python/python_apps.md]: Python apps can be launched in a variety of ways. You can specify how to start your application (for instance which module to run) by setting environment variables (in the "Environment variables" section of the console).

Something like that would be a bit better IMO

**Pull request merged**